props/dual_prop.h

```diff
@@ -3,31 +3,39 @@
 
 /*
 dual_prop.h allows for 2 discrete prop files to be used, 
-alternating on a latched switch (Blade Detect).
+alternating on a latched switch (Blade Detect) or with Blade ID..
 This is useful when you want a saber to 
 toggle to a blaster for example, 
 and you want the buttons to take on different behaviors.
 
 How to use:
+Edit your config file as follows.
+
+Add this to the CONFIG_TOP sction:
+#define DUAL_PROP_CONDITION (current_config != blades)
+
+Then setup the prop section like this:
 #ifdef CONFIG_PROP
 #include "../props/dual_prop.h"
 #include "../props/saber_sa22c_buttons.h"
 #include "../props/blaster.h"
 #undef PROP_TYPE
-#define PROP_TYPE DualProp<SaberSA22CButtons, Blaster>
+#define PROP_TYPE DualProp<SaberSA22CButtons, Blaster> // these are the classes found in the 2 props.
 #endif
-** Note the prop file SaberSA22CButtons here would change
-to the file name of the prop you choose.
-
-Then setup your config file for 2 states:
-Bladein preset bank, and a BladeOut preset bank.
-Give the BladeConfig 2 sets of descriptions, and
-#define BLADE_DETECT_PIN properly.
+** Note the prop file names (saber_sa22c_buttons.h and blaster.h)
+as well as the class names would change based on the prop you choose.
+
+Now setup your CONFIG_PRESETS section to have multiple preset banks.
```

Comment:
I think we should write this as two options: either use blade detect or blade id, then describe how to set up each option.

Reply:
done.
The POD will need updating next.